packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A common platform interface for the url_launcher plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.4
+version: 1.0.5
```

Comment:
You'll also want to update the dependency on plugin_platform_interface in this file.

Reply:
Good point, I'll have to either split this PR into 2, or follow-up with another PR just to bump after publishing the plugin_platform_interface. I'm going with the later as it's the faster path right now.

Reply:
Actually ended up splitting the PRs